yamllint/rules/new_lines.py

```diff
@@ -30,13 +34,14 @@
      type: unix
 """
 
+from os import linesep as os_linesep
```

Comment:
There is no need to rename the import here, please use:
```suggestion
from os import linesep
```

Reply:
I wanted to make it clear what this line-seperator is, namely that it's OS dependent. But if your finde with `linespe` I will use that